packages/reassure-cli/src/commands/measure.ts

```diff
@@ -118,30 +126,31 @@ export const command: CommandModule<{}, MeasureOptions> = {
   command: ['measure', '$0'],
   describe: 'Gather performance measurements by running performance tests',
   builder: (yargs) => {
-    return applyCommonOptions(yargs)
-      .option('baseline', {
+    return applyCommonOptions(yargs).options({
+      baseline: {
         type: 'boolean',
         default: false,
         describe: 'Save measurements as baseline instead of current',
-      })
-      .option('compare', {
+      },
+      compare: {
         type: 'boolean',
         default: true,
         describe: 'Outputs performance comparison results',
-      })
-      .option('branch', {
+      },
+      branch: {
         type: 'string',
         describe: 'Branch name of current code to be included in the report',
-      })
-      .option('commit-hash', {
+      },
+      commitHash: {
         type: 'string',
         describe: 'Commit hash of current code to be included in the report',
-      })
-      .option('testMatch', {
+      },
+      testMatch: {
```

Comment:
Seems we are getting into two conflicting option naming styles: `--testMatch` vs `--commit-hash`. @Xiltyn any thoughts on that.